### ISSUE TITLE
How do I run it?

### ISSUE THREAD

**Issue by robin8614** [iadgov/GRASSMARLIN#8]
I've read the .pdf and looked at the .pptx, and looked through the files. I must be missing something and feel dumb even asking, but how do I start GRASSMARLIN?

**Comment by iadgovuser5** [comment]
Once the program is installed on Windows, a "**GrassMarlin 3.2.1**" Shortcut should appear on the Desktop. Double-click this shortcut and the application should start.

If the shortcut does not get created for some reason, navigate to **C:\Program Files (x86)\IADGov\GrassMarlin3.2** and use the **GrassMarlin.bat** to lauch the application.

**Comment by widrlb** [comment]
Thanks for the response. I am interested in installing on Kali 2.0 (debian linux). My problem was that I couldn't find the Installers in the existing link for releases (https://github.com/iadgov/GRASSMARLIN/releases/tag/v3.2.1). I think the key to solving my question is this URL:  https://github.com/iadgov/GRASSMARLIN/releases and scrolling down to the Installers section.  I haven't had a chance to try it yet, but I assume once I download the appropriate .deb and install that, the rest will work as expected.

**Comment by robin8614** [comment]
Just in case this helps someone else:  I successfully installed and started Grassmarlin using the [Kali 2.0].deb and [Ubuntu-14-4].deb files from the "releases" link in the comment above. After downloading them, I ran dpkg -i [package].deb. Everything ran as expected then at the command line I typed: **grassmarlin**  and it started and the interface came up.

![grassmarlin_install_kali_run](https://user-images.githubusercontent.com/22416761/27813186-24385a86-6042-11e7-98ca-b44c51154772.PNG)


![lanching_grassmarlin](https://user-images.githubusercontent.com/22416761/27813163-edf3fc5a-6041-11e7-87b1-d9737138fb1d.PNG)

![grassmarlin_initial_interface](https://user-images.githubusercontent.com/22416761/27813169-00ebccac-6042-11e7-98bb-627bcc7f1404.PNG)

**Comment by jacksonp2008** [comment]
Yeah, not finding any documentation on the installer or configuration...  I did a standard dpkg -i on ubuntu, but when I start it sez:

```           
                        Launching GrassMarlin 3.2.1 ...   :kNK                
                                                             .                
                                                                                                                                           

GrassMarlin: An error has occurred. Refer to the GrassMarlin logs for a complete description.
```

So -- where are the logs, and where is the configuration file(s)?

I would assume, maybe incorrectly,  that GM will want to use a promiscuous mode ethernet port to sniff traffic on the wire?  (don't find that mentioned anywhere either)

Thanks

**Comment by jacksonp2008** [comment]
Stated no-where, GM requires ubuntu "desktop" and uses the native interface.  It is not webUI based as one would expect.

**Comment by IbarraSD** [comment]
Hello,
How to fix "Unable to initialize JNetCap" error
